### PR TITLE
Release 1.1.11

### PR DESCRIPTION
**Pull request by cmin764**
Release and publish a next package version given all the latest fixes.

1. Shall we increase the major/minor version instead? (since compatibility might break on Python 2)
2. Is it safe to `python -m build` and `twine upload dist/*` from a Mac system? (I don't see any Windows related compiled stuff in the main package)

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -1,6 +1,12 @@
 Comtypes CHANGELOG
 ==================
 
+Release 1.1.11
+-------------
+ * Fix Python 3 compatibility. Thanks @kdschlosser.
+ * Fix `tlib_mtime` in the absence of typelib.
+ * Fix 64bit inproc server.
```
Comment (cmin764) [inline CHANGES.txt:8]: Maybe we want to rephrase these, suggestions appreciated.

Comment (cfarrow) [reply CHANGES.txt:8]: I'm not sure what these refer to. I believe they date back to Thomas Heller.

**Comment by vasily-v-ryabov** [comment]
I'd suggest to review #241 first, because previous fix for timestamps is probably incomplete.

**Comment by cmin764** [comment]
Sounds good, please continue it from here, appreciate a lot your time & effort into this one! Let me know if you need any help with testing first.

**Review by vasily-v-ryabov — APPROVED** [review]

**Pull request merged**